docs/contributing/editing-markdown.md

```diff
@@ -1,4 +1,4 @@
-### Editing Pages (Markdown Files)
```

Comment:
@samajammin @minimalsm I renamed this and the next file that was in uppercase to lowercase (kebab) for consistency. I have searched in the repo if we were linking these files from somewhere but I couldn't find anything.
Do you know if we have some external resources linking to any of these files?

Reply:
No I think we're fine 👍